examples/calibrate_your_rule.py

```diff
@@ -62,9 +55,11 @@ def test_example_ways_to_use_infertrade_2():
 
     # Pipeline version
     rule_plus_returns = make_pipeline(buy_on_small_rises_rule, returns_calc)
-    my_dataframe_with_returns_2 = rule_plus_returns(my_dataframe_without_allocations)
+    my_dataframe_with_returns_2 = rule_plus_returns.fit_transform(my_dataframe_without_allocations)
 
-    assert (my_dataframe_with_returns == my_dataframe_with_returns_2).all()
+    comparison = (my_dataframe_with_returns == my_dataframe_with_returns_2)
+    comparison[pd.isnull(my_dataframe_with_returns) & pd.isnull(my_dataframe_with_returns_2)] = True
+    assert comparison.values.all()
 
```

Comment:
NaN == NaN results in False. As per https://stackoverflow.com/questions/19322506/pandas-dataframes-with-nans-equality-comparison .